client/src/components/squids/SquidShow.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC } from "react";
+
+import { compareAsc, format, parseISO } from "date-fns";
+import { useParams } from "react-router-dom";
+
+import { useSquidDetails } from "./hooks/useSquidDetails";
+import { SquidSign } from "./SquidSign";
+
+import "../../style/squids/SquidShow.pcss";
+
+export const SquidShow: FC = () => {
+  const { id } = useParams<{ id: string }>();
+
+  const { data: squid, isLoading, isSuccess } = useSquidDetails(id);
+
+  if (isLoading) {
+    return (
+      <h3 className="squid-show squid__heading">One moment while the squid is swimming over...</h3>
+    );
+  }
+
+  if (isSuccess && squid) {
+    const greeting = (): string => {
```

Comment:
This would be great to move into a custom hook! One of the advantages of custom hooks is that they let us go back to a React 15 way of thinking about separating presentational and computational concerns (kinda. a bit.). 

One thing to anticipate -- you can't execute a hook conditionally, so you'd need to execute the hook above the conditional even if you have no squid data, then have it return null if there's no squid data. OR, you could extract out a <SquidGreeting/> component and put the hook in there -- since you'd only render SquidGreeting if you've got a squid, you wouldn't have to worry about the hook running needlessly.